### PR TITLE
Update phoenix guides for v1.3 specifics

### PR DESCRIPTION
**Pull request by untra**
* Updates the phoenix documentation with specifics regarding Phoenix v1.3 production deployment configuration
* Demo here: https://github.com/untra/hello

### CODE REVIEW

**Comment by tylerflint** [comment]
@untra This is great stuff! Thanks.

@sanderson, @sdomino: Can we review this?

File: articles/elixir/phoenix/launch-your-app.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## Add a Hosting Provider
+Nanobox provisions and deploys your app on your hosting provider. To link a hosting account, go to your account admin under <a href="https://dashboard.nanobox.io/users/provider_accounts" target="\_blank">hosting accounts</a>.
+
+## Launch a New App
+New apps are launched from your <a href="https://dashboard.nanobox.io" target="\_blank">app dashboard</a>.
+
+#### Phoenix 1.3 PORT environment variable
+
+In Phoenix v1.3 and up, the `PORT` environment variable must be set when running when in production.
```
Comment (untra) [inline articles/elixir/phoenix/launch-your-app.md]: > when running when in production.

fudge, I just saw this. Let me fix my typo.

**Comment by untra** [comment]
Okay, I think it looks good now 👍

**Review by sanderson — COMMENTED** [review]
Everything looks great. I just notice a double "with" `with with` on line 3 of launch-your-app.md. If you want to push a quick fix for that, I'll approve it and merge it in.

**Comment by untra** [comment]
I clearly can't spell, haha!
Thanks for the eyes. I've updated things. If it looks good, merge when ready.
nanobox is rad 👍

**Comment by sanderson** [comment]
Awesome! Thanks! Merged.

**Pull request merged**